scripts/nlp_api_master.py

```diff
@@ -43,6 +43,25 @@
                         "keras_nlp.models.AlbertClassifier.preprocessor",
                     ],
                 },
+                {
+                    "path": "albert_masked_lm",
```

Comment:
I think we added a documented property `token_embedding` for all backbones. Should we add that too?